diff --git a/myapp/build.gradle b/myapp/build.gradle
index eb29f2b..e2cb3f7 100644
--- a/myapp/build.gradle
+++ b/myapp/build.gradle
@@ -5,7 +5,7 @@ buildscript {
     dependencies {
         classpath "org.grails:grails-gradle-plugin:$grailsVersion"
         classpath "org.grails.plugins:hibernate5:7.0.0"
-        classpath "org.grails.plugins:views-gradle:2.0.0.BUILD-SNAPSHOT"
+        classpath "org.grails.plugins:views-gradle:2.0.0.RC2"
     }
 }
 
@@ -66,6 +66,7 @@ dependencies {
 }
 
 bootRun {
+    ignoreExitValue true
     jvmArgs(
         '-Dspring.output.ansi.enabled=always', 
         '-noverify', 
@@ -75,3 +76,9 @@ bootRun {
     String springProfilesActive = 'spring.profiles.active'
     systemProperty springProfilesActive, System.getProperty(springProfilesActive)
 }
+
+tasks.withType(GroovyCompile) {
+    configure(groovyOptions) {
+        forkOptions.jvmArgs = ['-Xmx1024m']
+    }
+}
diff --git a/myapp/gradle.properties b/myapp/gradle.properties
index 73b7708..04c3976 100644
--- a/myapp/gradle.properties
+++ b/myapp/gradle.properties
@@ -1,2 +1,5 @@
-grailsVersion=4.0.0
-gorm.version=7.0.2.RELEASE
+grailsVersion=4.0.2
+gorm.version=7.0.3.RELEASE
+org.gradle.daemon=true
+org.gradle.parallel=true
+org.gradle.jvmargs=-Dfile.encoding=UTF-8 -Xmx1024M
