diff --git a/myapp/build.gradle b/myapp/build.gradle
index eb29f2be..d31ad074 100644
--- a/myapp/build.gradle
+++ b/myapp/build.gradle
@@ -4,8 +4,8 @@ buildscript {
     }
     dependencies {
         classpath "org.grails:grails-gradle-plugin:$grailsVersion"
-        classpath "org.grails.plugins:hibernate5:7.0.0"
-        classpath "org.grails.plugins:views-gradle:2.0.0.BUILD-SNAPSHOT"
+        classpath "org.grails.plugins:hibernate5:7.0.4"
+        classpath "org.grails.plugins:views-gradle:2.0.2"
     }
 }
 
@@ -49,7 +49,7 @@ dependencies {
     compile "org.grails.plugins:async"
     compile "org.grails.plugins:events"
     compile "org.grails.plugins:hibernate5"
-    compile "org.hibernate:hibernate-core:5.4.0.Final"
+    compile "org.hibernate:hibernate-core:5.4.18.Final"
     compile "org.grails.plugins:views-json"
     compile "org.grails.plugins:views-json-templates"
     compileOnly "io.micronaut:micronaut-inject-groovy"
@@ -58,14 +58,18 @@ dependencies {
     runtime "org.glassfish.web:el-impl:2.1.2-b03"
     runtime "com.h2database:h2"
     runtime "org.apache.tomcat:tomcat-jdbc"
-    runtime "javax.xml.bind:jaxb-api:2.3.0"
+    runtime "javax.xml.bind:jaxb-api:2.3.1"
+    testCompile "io.micronaut:micronaut-inject-groovy"
     testCompile "org.grails:grails-gorm-testing-support"
     testCompile "org.mockito:mockito-core"
     testCompile "io.micronaut:micronaut-http-client"
     testCompile "org.grails:grails-web-testing-support"
+    testCompile "org.grails:views-json-testing-support"
+    testCompile "org.hibernate.validator:hibernate-validator:6.0.20.Final"
 }
 
 bootRun {
+    ignoreExitValue true
     jvmArgs(
         '-Dspring.output.ansi.enabled=always', 
         '-noverify', 
@@ -75,3 +79,9 @@ bootRun {
     String springProfilesActive = 'spring.profiles.active'
     systemProperty springProfilesActive, System.getProperty(springProfilesActive)
 }
+
+tasks.withType(GroovyCompile) {
+    configure(groovyOptions) {
+        forkOptions.jvmArgs = ['-Xmx1024m']
+    }
+}
diff --git a/myapp/gradle.properties b/myapp/gradle.properties
index 73b7708c..39337b17 100644
--- a/myapp/gradle.properties
+++ b/myapp/gradle.properties
@@ -1,2 +1,5 @@
-grailsVersion=4.0.0
-gorm.version=7.0.2.RELEASE
+grailsVersion=4.0.8
+gorm.version=7.0.8.RELEASE
+org.gradle.daemon=true
+org.gradle.parallel=true
+org.gradle.jvmargs=-Dfile.encoding=UTF-8 -Xmx1024M
diff --git a/myapp/gradle/wrapper/gradle-wrapper.properties b/myapp/gradle/wrapper/gradle-wrapper.properties
index 558870da..5028f28f 100644
--- a/myapp/gradle/wrapper/gradle-wrapper.properties
+++ b/myapp/gradle/wrapper/gradle-wrapper.properties
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.1.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-bin.zip
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
