diff --git a/myapp/build.gradle b/myapp/build.gradle
index 54a8355..d14cff8 100644
--- a/myapp/build.gradle
+++ b/myapp/build.gradle
@@ -18,8 +18,8 @@ apply plugin:"idea"
 apply plugin:"war"
 apply plugin:"org.grails.grails-web"
 apply plugin:"com.github.erdi.webdriver-binaries"
-apply plugin:"org.grails.grails-gsp"
 apply plugin:"com.bertramlabs.asset-pipeline"
+apply plugin:"org.grails.grails-gsp"
 
 repositories {
     maven { url "https://repo.grails.org/grails/core" }
@@ -68,11 +68,11 @@ dependencies {
     testImplementation "org.mockito:mockito-core"
     testImplementation "org.grails:grails-web-testing-support"
     testImplementation "org.grails.plugins:geb"
-    testImplementation "org.seleniumhq.selenium:selenium-remote-driver:3.141.59"
-    testImplementation "org.seleniumhq.selenium:selenium-api:3.141.59"
-    testImplementation "org.seleniumhq.selenium:selenium-support:3.141.59"
-    testRuntimeOnly "org.seleniumhq.selenium:selenium-chrome-driver:3.141.59"
-    testRuntimeOnly "org.seleniumhq.selenium:selenium-firefox-driver:3.141.59"
+    testImplementation "org.seleniumhq.selenium:selenium-remote-driver:4.0.0"
+    testImplementation "org.seleniumhq.selenium:selenium-api:4.0.0"
+    testImplementation "org.seleniumhq.selenium:selenium-support:4.0.0"
+    testRuntimeOnly "org.seleniumhq.selenium:selenium-chrome-driver:4.0.0"
+    testRuntimeOnly "org.seleniumhq.selenium:selenium-firefox-driver:4.0.0"
 }
 
 bootRun {
@@ -99,7 +99,7 @@ tasks.withType(Test) {
 
 webdriverBinaries {
     chromedriver '2.45.0'
-    geckodriver '0.24.0'
+    geckodriver '0.30.0'
 }
 
 tasks.withType(Test) {
diff --git a/myapp/gradle.properties b/myapp/gradle.properties
index 47997f8..d85f348 100644
--- a/myapp/gradle.properties
+++ b/myapp/gradle.properties
@@ -1,4 +1,4 @@
-grailsVersion=5.0.2
+grailsVersion=5.0.3
 grailsGradlePluginVersion=5.0.0
 groovyVersion=3.0.7
 gorm.version=7.1.0
diff --git a/myapp/grails-app/conf/application.yml b/myapp/grails-app/conf/application.yml
index 754df0b..3d3c6a8 100644
--- a/myapp/grails-app/conf/application.yml
+++ b/myapp/grails-app/conf/application.yml
@@ -86,10 +86,6 @@ grails:
                 scriptlet: html
                 taglib: none
                 staticparts: none
-management:
-    endpoints:
-        jmx:
-            unique-names: true
 
 ---
 hibernate:
