diff --git a/myplugin/build.gradle b/myplugin/build.gradle
index 9d93086..9d2bfcb 100644
--- a/myplugin/build.gradle
+++ b/myplugin/build.gradle
@@ -4,8 +4,8 @@ buildscript {
     }
     dependencies {
         classpath "org.grails:grails-gradle-plugin:$grailsGradlePluginVersion"
-        classpath "org.grails.plugins:hibernate5:7.1.0"
-        classpath "org.grails.plugins:views-gradle:2.1.1"
+        classpath "org.grails.plugins:hibernate5:7.2.0"
+        classpath "org.grails.plugins:views-gradle:2.1.2"
     }
 }
 
@@ -50,9 +50,9 @@ dependencies {
     implementation "org.grails:grails-logging"
     implementation "org.grails.plugins:cache"
     implementation "org.grails.plugins:async"
-    implementation "org.grails.plugins:hibernate5"
-    implementation "org.hibernate:hibernate-core:5.5.7.Final"
     implementation "org.grails.plugins:events"
+    implementation "org.grails.plugins:hibernate5"
+    implementation "org.hibernate:hibernate-core:5.6.3.Final"
     implementation "org.grails.plugins:views-json"
     implementation "org.grails.plugins:views-json-templates"
     profile "org.grails.profiles:rest-api"
diff --git a/myplugin/gradle.properties b/myplugin/gradle.properties
index d85f348..c83370a 100644
--- a/myplugin/gradle.properties
+++ b/myplugin/gradle.properties
@@ -1,7 +1,7 @@
-grailsVersion=5.0.3
-grailsGradlePluginVersion=5.0.0
+grailsVersion=5.1.1
+grailsGradlePluginVersion=5.1.0
 groovyVersion=3.0.7
-gorm.version=7.1.0
+gorm.version=7.1.2
 org.gradle.daemon=true
 org.gradle.parallel=true
 org.gradle.jvmargs=-Dfile.encoding=UTF-8 -Xmx1024M
diff --git a/myplugin/settings.gradle b/myplugin/settings.gradle
new file mode 100644
index 0000000..8d5eb67
--- /dev/null
+++ b/myplugin/settings.gradle
@@ -0,0 +1 @@
+rootProject.name = 'myplugin'
