diff --git a/myapp/settings.gradle b/myapp/settings.gradle
deleted file mode 100644
index 6d35791..0000000
--- a/myapp/settings.gradle
+++ /dev/null
@@ -1 +0,0 @@
-rootProject.name = 'myapp'
diff --git a/myapp/.gitignore b/myplugin/.gitignore
similarity index 100%
rename from myapp/.gitignore
rename to myplugin/.gitignore
diff --git a/myapp/build.gradle b/myplugin/build.gradle
similarity index 94%
rename from myapp/build.gradle
rename to myplugin/build.gradle
index e2f0447..5d31823 100644
--- a/myapp/build.gradle
+++ b/myplugin/build.gradle
@@ -4,13 +4,13 @@ buildscript {
     }
     dependencies {
         classpath "org.grails:grails-gradle-plugin:$grailsGradlePluginVersion"
-        classpath "org.grails.plugins:hibernate5:7.2.2"
-        classpath "org.grails.plugins:views-gradle:2.2.1"
+        classpath "org.grails.plugins:hibernate5:7.3.0"
+        classpath "org.grails.plugins:views-gradle:2.3.2"
     }
 }
 
 version "0.1"
-group "myapp"
+group "myplugin"
 
 apply plugin:"eclipse"
 apply plugin:"idea"
@@ -50,9 +50,9 @@ dependencies {
     implementation "org.grails:grails-logging"
     implementation "org.grails.plugins:cache"
     implementation "org.grails.plugins:async"
-    implementation "org.grails.plugins:hibernate5"
-    implementation "org.hibernate:hibernate-core:5.6.5.Final"
     implementation "org.grails.plugins:events"
+    implementation "org.grails.plugins:hibernate5"
+    implementation "org.hibernate:hibernate-core:5.6.9.Final"
     implementation "org.grails.plugins:views-json"
     implementation "org.grails.plugins:views-json-templates"
     profile "org.grails.profiles:rest-api"
diff --git a/myapp/gradle.properties b/myplugin/gradle.properties
similarity index 51%
rename from myapp/gradle.properties
rename to myplugin/gradle.properties
index 2b3b5d7..884b089 100644
--- a/myapp/gradle.properties
+++ b/myplugin/gradle.properties
@@ -1,7 +1,7 @@
-grailsVersion=5.1.4
-grailsGradlePluginVersion=5.1.3
-groovyVersion=3.0.7
-gorm.version=7.2.1
+grailsVersion=5.2.2
+grailsGradlePluginVersion=5.2.2
+groovyVersion=3.0.11
+gorm.version=7.3.2
 org.gradle.daemon=true
 org.gradle.parallel=true
 org.gradle.jvmargs=-Dfile.encoding=UTF-8 -Xmx1024M
diff --git a/myapp/gradle/wrapper/gradle-wrapper.jar b/myplugin/gradle/wrapper/gradle-wrapper.jar
similarity index 100%
rename from myapp/gradle/wrapper/gradle-wrapper.jar
rename to myplugin/gradle/wrapper/gradle-wrapper.jar
diff --git a/myapp/gradle/wrapper/gradle-wrapper.properties b/myplugin/gradle/wrapper/gradle-wrapper.properties
similarity index 100%
rename from myapp/gradle/wrapper/gradle-wrapper.properties
rename to myplugin/gradle/wrapper/gradle-wrapper.properties
diff --git a/myapp/gradlew b/myplugin/gradlew
similarity index 100%
rename from myapp/gradlew
rename to myplugin/gradlew
diff --git a/myapp/gradlew.bat b/myplugin/gradlew.bat
similarity index 100%
rename from myapp/gradlew.bat
rename to myplugin/gradlew.bat
diff --git a/myapp/grails-app/conf/application.yml b/myplugin/grails-app/conf/application.yml
similarity index 99%
rename from myapp/grails-app/conf/application.yml
rename to myplugin/grails-app/conf/application.yml
index b74239d..b3ce9b8 100644
--- a/myapp/grails-app/conf/application.yml
+++ b/myplugin/grails-app/conf/application.yml
@@ -2,7 +2,7 @@
 grails:
     profile: rest-api
     codegen:
-        defaultPackage: myapp
+        defaultPackage: myplugin
     gorm:
         reactor:
             # Whether to translate GORM events into Reactor events
diff --git a/myapp/grails-app/conf/logback.xml b/myplugin/grails-app/conf/logback.xml
similarity index 100%
rename from myapp/grails-app/conf/logback.xml
rename to myplugin/grails-app/conf/logback.xml
diff --git a/myapp/grails-app/conf/spring/resources.groovy b/myplugin/grails-app/conf/spring/resources.groovy
similarity index 100%
rename from myapp/grails-app/conf/spring/resources.groovy
rename to myplugin/grails-app/conf/spring/resources.groovy
diff --git a/myapp/grails-app/controllers/myapp/ApplicationController.groovy b/myplugin/grails-app/controllers/myplugin/ApplicationController.groovy
similarity index 94%
rename from myapp/grails-app/controllers/myapp/ApplicationController.groovy
rename to myplugin/grails-app/controllers/myplugin/ApplicationController.groovy
index d40d868..bc28483 100644
--- a/myapp/grails-app/controllers/myapp/ApplicationController.groovy
+++ b/myplugin/grails-app/controllers/myplugin/ApplicationController.groovy
@@ -1,4 +1,4 @@
-package myapp
+package myplugin
 
 import grails.core.GrailsApplication
 import grails.plugins.*
diff --git a/myapp/grails-app/controllers/myapp/UrlMappings.groovy b/myplugin/grails-app/controllers/myplugin/UrlMappings.groovy
similarity index 96%
rename from myapp/grails-app/controllers/myapp/UrlMappings.groovy
rename to myplugin/grails-app/controllers/myplugin/UrlMappings.groovy
index 34bcc50..2b44064 100644
--- a/myapp/grails-app/controllers/myapp/UrlMappings.groovy
+++ b/myplugin/grails-app/controllers/myplugin/UrlMappings.groovy
@@ -1,4 +1,4 @@
-package myapp
+package myplugin
 
 class UrlMappings {
 
diff --git a/myapp/grails-app/i18n/messages.properties b/myplugin/grails-app/i18n/messages.properties
similarity index 100%
rename from myapp/grails-app/i18n/messages.properties
rename to myplugin/grails-app/i18n/messages.properties
diff --git a/myapp/grails-app/i18n/messages_cs.properties b/myplugin/grails-app/i18n/messages_cs.properties
similarity index 100%
rename from myapp/grails-app/i18n/messages_cs.properties
rename to myplugin/grails-app/i18n/messages_cs.properties
diff --git a/myapp/grails-app/i18n/messages_da.properties b/myplugin/grails-app/i18n/messages_da.properties
similarity index 100%
rename from myapp/grails-app/i18n/messages_da.properties
rename to myplugin/grails-app/i18n/messages_da.properties
diff --git a/myapp/grails-app/i18n/messages_de.properties b/myplugin/grails-app/i18n/messages_de.properties
similarity index 100%
rename from myapp/grails-app/i18n/messages_de.properties
rename to myplugin/grails-app/i18n/messages_de.properties
diff --git a/myapp/grails-app/i18n/messages_es.properties b/myplugin/grails-app/i18n/messages_es.properties
similarity index 100%
rename from myapp/grails-app/i18n/messages_es.properties
rename to myplugin/grails-app/i18n/messages_es.properties
diff --git a/myapp/grails-app/i18n/messages_fr.properties b/myplugin/grails-app/i18n/messages_fr.properties
similarity index 100%
rename from myapp/grails-app/i18n/messages_fr.properties
rename to myplugin/grails-app/i18n/messages_fr.properties
diff --git a/myapp/grails-app/i18n/messages_it.properties b/myplugin/grails-app/i18n/messages_it.properties
similarity index 100%
rename from myapp/grails-app/i18n/messages_it.properties
rename to myplugin/grails-app/i18n/messages_it.properties
diff --git a/myapp/grails-app/i18n/messages_ja.properties b/myplugin/grails-app/i18n/messages_ja.properties
similarity index 100%
rename from myapp/grails-app/i18n/messages_ja.properties
rename to myplugin/grails-app/i18n/messages_ja.properties
diff --git a/myapp/grails-app/i18n/messages_nb.properties b/myplugin/grails-app/i18n/messages_nb.properties
similarity index 100%
rename from myapp/grails-app/i18n/messages_nb.properties
rename to myplugin/grails-app/i18n/messages_nb.properties
diff --git a/myapp/grails-app/i18n/messages_nl.properties b/myplugin/grails-app/i18n/messages_nl.properties
similarity index 100%
rename from myapp/grails-app/i18n/messages_nl.properties
rename to myplugin/grails-app/i18n/messages_nl.properties
diff --git a/myapp/grails-app/i18n/messages_pl.properties b/myplugin/grails-app/i18n/messages_pl.properties
similarity index 100%
rename from myapp/grails-app/i18n/messages_pl.properties
rename to myplugin/grails-app/i18n/messages_pl.properties
diff --git a/myapp/grails-app/i18n/messages_pt_BR.properties b/myplugin/grails-app/i18n/messages_pt_BR.properties
similarity index 100%
rename from myapp/grails-app/i18n/messages_pt_BR.properties
rename to myplugin/grails-app/i18n/messages_pt_BR.properties
diff --git a/myapp/grails-app/i18n/messages_pt_PT.properties b/myplugin/grails-app/i18n/messages_pt_PT.properties
similarity index 100%
rename from myapp/grails-app/i18n/messages_pt_PT.properties
rename to myplugin/grails-app/i18n/messages_pt_PT.properties
diff --git a/myapp/grails-app/i18n/messages_ru.properties b/myplugin/grails-app/i18n/messages_ru.properties
similarity index 100%
rename from myapp/grails-app/i18n/messages_ru.properties
rename to myplugin/grails-app/i18n/messages_ru.properties
diff --git a/myapp/grails-app/i18n/messages_sk.properties b/myplugin/grails-app/i18n/messages_sk.properties
similarity index 100%
rename from myapp/grails-app/i18n/messages_sk.properties
rename to myplugin/grails-app/i18n/messages_sk.properties
diff --git a/myapp/grails-app/i18n/messages_sv.properties b/myplugin/grails-app/i18n/messages_sv.properties
similarity index 100%
rename from myapp/grails-app/i18n/messages_sv.properties
rename to myplugin/grails-app/i18n/messages_sv.properties
diff --git a/myapp/grails-app/i18n/messages_th.properties b/myplugin/grails-app/i18n/messages_th.properties
similarity index 100%
rename from myapp/grails-app/i18n/messages_th.properties
rename to myplugin/grails-app/i18n/messages_th.properties
diff --git a/myapp/grails-app/i18n/messages_zh_CN.properties b/myplugin/grails-app/i18n/messages_zh_CN.properties
similarity index 100%
rename from myapp/grails-app/i18n/messages_zh_CN.properties
rename to myplugin/grails-app/i18n/messages_zh_CN.properties
diff --git a/myapp/grails-app/init/myapp/Application.groovy b/myplugin/grails-app/init/myplugin/Application.groovy
similarity index 93%
rename from myapp/grails-app/init/myapp/Application.groovy
rename to myplugin/grails-app/init/myplugin/Application.groovy
index 980180d..ace04f1 100644
--- a/myapp/grails-app/init/myapp/Application.groovy
+++ b/myplugin/grails-app/init/myplugin/Application.groovy
@@ -1,4 +1,4 @@
-package myapp
+package myplugin
 
 import grails.boot.GrailsApp
 import grails.boot.config.GrailsAutoConfiguration
diff --git a/myapp/grails-app/init/myapp/BootStrap.groovy b/myplugin/grails-app/init/myplugin/BootStrap.groovy
similarity index 83%
rename from myapp/grails-app/init/myapp/BootStrap.groovy
rename to myplugin/grails-app/init/myplugin/BootStrap.groovy
index 5cf149b..e8ad49b 100644
--- a/myapp/grails-app/init/myapp/BootStrap.groovy
+++ b/myplugin/grails-app/init/myplugin/BootStrap.groovy
@@ -1,4 +1,4 @@
-package myapp
+package myplugin
 
 class BootStrap {
 
diff --git a/myapp/grails-app/views/application/index.gson b/myplugin/grails-app/views/application/index.gson
similarity index 100%
rename from myapp/grails-app/views/application/index.gson
rename to myplugin/grails-app/views/application/index.gson
diff --git a/myapp/grails-app/views/error.gson b/myplugin/grails-app/views/error.gson
similarity index 100%
rename from myapp/grails-app/views/error.gson
rename to myplugin/grails-app/views/error.gson
diff --git a/myapp/grails-app/views/errors/_errors.gson b/myplugin/grails-app/views/errors/_errors.gson
similarity index 100%
rename from myapp/grails-app/views/errors/_errors.gson
rename to myplugin/grails-app/views/errors/_errors.gson
diff --git a/myapp/grails-app/views/notFound.gson b/myplugin/grails-app/views/notFound.gson
similarity index 100%
rename from myapp/grails-app/views/notFound.gson
rename to myplugin/grails-app/views/notFound.gson
diff --git a/myapp/grails-app/views/object/_object.gson b/myplugin/grails-app/views/object/_object.gson
similarity index 100%
rename from myapp/grails-app/views/object/_object.gson
rename to myplugin/grails-app/views/object/_object.gson
diff --git a/myapp/grails-wrapper.jar b/myplugin/grails-wrapper.jar
similarity index 100%
rename from myapp/grails-wrapper.jar
rename to myplugin/grails-wrapper.jar
diff --git a/myapp/grailsw b/myplugin/grailsw
similarity index 100%
rename from myapp/grailsw
rename to myplugin/grailsw
diff --git a/myapp/grailsw.bat b/myplugin/grailsw.bat
similarity index 100%
rename from myapp/grailsw.bat
rename to myplugin/grailsw.bat
diff --git a/myplugin/settings.gradle b/myplugin/settings.gradle
new file mode 100644
index 0000000..8d5eb67
--- /dev/null
+++ b/myplugin/settings.gradle
@@ -0,0 +1 @@
+rootProject.name = 'myplugin'
