diff --git a/myplugin/build.gradle b/myplugin/build.gradle
index 0ca95909..9086fb4a 100644
--- a/myplugin/build.gradle
+++ b/myplugin/build.gradle
@@ -1,6 +1,5 @@
 buildscript {
     repositories {
-        maven { url "https://plugins.gradle.org/m2/" }
         maven { url "https://repo.grails.org/grails/core" }
     }
     dependencies {
@@ -19,7 +18,6 @@ apply plugin:"asset-pipeline"
 apply plugin:"org.grails.grails-gsp"
 
 repositories {
-    mavenCentral()
     maven { url "https://repo.grails.org/grails/core" }
 }
 
diff --git a/myplugin/gradle.properties b/myplugin/gradle.properties
index e7a8ca74..884b0893 100644
--- a/myplugin/gradle.properties
+++ b/myplugin/gradle.properties
@@ -1,5 +1,5 @@
-grailsVersion=5.2.5
-grailsGradlePluginVersion=5.2.4
+grailsVersion=5.2.2
+grailsGradlePluginVersion=5.2.2
 groovyVersion=3.0.11
 gorm.version=7.3.2
 org.gradle.daemon=true
diff --git a/myplugin/src/main/groovy/myplugin/MypluginGrailsPlugin.groovy b/myplugin/src/main/groovy/myplugin/MypluginGrailsPlugin.groovy
index 52473ca7..40d95834 100644
--- a/myplugin/src/main/groovy/myplugin/MypluginGrailsPlugin.groovy
+++ b/myplugin/src/main/groovy/myplugin/MypluginGrailsPlugin.groovy
@@ -5,7 +5,7 @@ import grails.plugins.*
 class MypluginGrailsPlugin extends Plugin {
 
     // the version or versions of Grails the plugin is designed for
-    def grailsVersion = "5.2.5 > *"
+    def grailsVersion = "5.2.2 > *"
     // resources that are excluded from plugin packaging
     def pluginExcludes = [
         "grails-app/views/error.gsp"
