diff --git a/myplugin/build.gradle b/myplugin/build.gradle
index 9086fb4a..0ca95909 100644
--- a/myplugin/build.gradle
+++ b/myplugin/build.gradle
@@ -1,5 +1,6 @@
 buildscript {
     repositories {
+        maven { url "https://plugins.gradle.org/m2/" }
         maven { url "https://repo.grails.org/grails/core" }
     }
     dependencies {
@@ -18,6 +19,7 @@ apply plugin:"asset-pipeline"
 apply plugin:"org.grails.grails-gsp"
 
 repositories {
+    mavenCentral()
     maven { url "https://repo.grails.org/grails/core" }
 }
 
diff --git a/myplugin/gradle.properties b/myplugin/gradle.properties
index 2cc0d8a5..611c300c 100644
--- a/myplugin/gradle.properties
+++ b/myplugin/gradle.properties
@@ -1,7 +1,7 @@
-grailsVersion=5.2.4
-grailsGradlePluginVersion=5.2.3
+grailsVersion=5.3.2
+grailsGradlePluginVersion=5.3.0
 groovyVersion=3.0.11
-gorm.version=7.3.2
+gorm.version=7.3.3
 org.gradle.daemon=true
 org.gradle.parallel=true
 org.gradle.jvmargs=-Dfile.encoding=UTF-8 -Xmx1024M
diff --git a/myplugin/src/main/groovy/myplugin/MypluginGrailsPlugin.groovy b/myplugin/src/main/groovy/myplugin/MypluginGrailsPlugin.groovy
index adfb1e70..211b04fe 100644
--- a/myplugin/src/main/groovy/myplugin/MypluginGrailsPlugin.groovy
+++ b/myplugin/src/main/groovy/myplugin/MypluginGrailsPlugin.groovy
@@ -5,7 +5,7 @@ import grails.plugins.*
 class MypluginGrailsPlugin extends Plugin {
 
     // the version or versions of Grails the plugin is designed for
-    def grailsVersion = "5.2.4 > *"
+    def grailsVersion = "5.3.2 > *"
     // resources that are excluded from plugin packaging
     def pluginExcludes = [
         "grails-app/views/error.gsp"
