diff --git a/myplugin/.gitignore b/myapp/.gitignore
similarity index 100%
rename from myplugin/.gitignore
rename to myapp/.gitignore
diff --git a/myplugin/build.gradle b/myapp/build.gradle
similarity index 99%
rename from myplugin/build.gradle
rename to myapp/build.gradle
index b4ada81..d838ef0 100644
--- a/myplugin/build.gradle
+++ b/myapp/build.gradle
@@ -11,7 +11,7 @@ buildscript {
 }
 
 version "0.1"
-group "myplugin"
+group "myapp"
 
 apply plugin:"eclipse"
 apply plugin:"idea"
diff --git a/myplugin/gradle.properties b/myapp/gradle.properties
similarity index 72%
rename from myplugin/gradle.properties
rename to myapp/gradle.properties
index a8dcc92..884b089 100644
--- a/myplugin/gradle.properties
+++ b/myapp/gradle.properties
@@ -1,5 +1,5 @@
-grailsVersion=5.2.1
-grailsGradlePluginVersion=5.2.1
+grailsVersion=5.2.2
+grailsGradlePluginVersion=5.2.2
 groovyVersion=3.0.11
 gorm.version=7.3.2
 org.gradle.daemon=true
diff --git a/myplugin/gradle/wrapper/gradle-wrapper.jar b/myapp/gradle/wrapper/gradle-wrapper.jar
similarity index 100%
rename from myplugin/gradle/wrapper/gradle-wrapper.jar
rename to myapp/gradle/wrapper/gradle-wrapper.jar
diff --git a/myplugin/gradle/wrapper/gradle-wrapper.properties b/myapp/gradle/wrapper/gradle-wrapper.properties
similarity index 100%
rename from myplugin/gradle/wrapper/gradle-wrapper.properties
rename to myapp/gradle/wrapper/gradle-wrapper.properties
diff --git a/myplugin/gradlew b/myapp/gradlew
similarity index 100%
rename from myplugin/gradlew
rename to myapp/gradlew
diff --git a/myplugin/gradlew.bat b/myapp/gradlew.bat
similarity index 100%
rename from myplugin/gradlew.bat
rename to myapp/gradlew.bat
diff --git a/myplugin/grails-app/assets/images/advancedgrails.svg b/myapp/grails-app/assets/images/advancedgrails.svg
similarity index 100%
rename from myplugin/grails-app/assets/images/advancedgrails.svg
rename to myapp/grails-app/assets/images/advancedgrails.svg
diff --git a/myplugin/grails-app/assets/images/apple-touch-icon-retina.png b/myapp/grails-app/assets/images/apple-touch-icon-retina.png
similarity index 100%
rename from myplugin/grails-app/assets/images/apple-touch-icon-retina.png
rename to myapp/grails-app/assets/images/apple-touch-icon-retina.png
diff --git a/myplugin/grails-app/assets/images/apple-touch-icon.png b/myapp/grails-app/assets/images/apple-touch-icon.png
similarity index 100%
rename from myplugin/grails-app/assets/images/apple-touch-icon.png
rename to myapp/grails-app/assets/images/apple-touch-icon.png
diff --git a/myplugin/grails-app/assets/images/documentation.svg b/myapp/grails-app/assets/images/documentation.svg
similarity index 100%
rename from myplugin/grails-app/assets/images/documentation.svg
rename to myapp/grails-app/assets/images/documentation.svg
diff --git a/myplugin/grails-app/assets/images/favicon.ico b/myapp/grails-app/assets/images/favicon.ico
similarity index 100%
rename from myplugin/grails-app/assets/images/favicon.ico
rename to myapp/grails-app/assets/images/favicon.ico
diff --git a/myplugin/grails-app/assets/images/grails-cupsonly-logo-white.svg b/myapp/grails-app/assets/images/grails-cupsonly-logo-white.svg
similarity index 100%
rename from myplugin/grails-app/assets/images/grails-cupsonly-logo-white.svg
rename to myapp/grails-app/assets/images/grails-cupsonly-logo-white.svg
diff --git a/myplugin/grails-app/assets/images/grails.svg b/myapp/grails-app/assets/images/grails.svg
similarity index 100%
rename from myplugin/grails-app/assets/images/grails.svg
rename to myapp/grails-app/assets/images/grails.svg
diff --git a/myplugin/grails-app/assets/images/skin/database_add.png b/myapp/grails-app/assets/images/skin/database_add.png
similarity index 100%
rename from myplugin/grails-app/assets/images/skin/database_add.png
rename to myapp/grails-app/assets/images/skin/database_add.png
diff --git a/myplugin/grails-app/assets/images/skin/database_delete.png b/myapp/grails-app/assets/images/skin/database_delete.png
similarity index 100%
rename from myplugin/grails-app/assets/images/skin/database_delete.png
rename to myapp/grails-app/assets/images/skin/database_delete.png
diff --git a/myplugin/grails-app/assets/images/skin/database_edit.png b/myapp/grails-app/assets/images/skin/database_edit.png
similarity index 100%
rename from myplugin/grails-app/assets/images/skin/database_edit.png
rename to myapp/grails-app/assets/images/skin/database_edit.png
diff --git a/myplugin/grails-app/assets/images/skin/database_save.png b/myapp/grails-app/assets/images/skin/database_save.png
similarity index 100%
rename from myplugin/grails-app/assets/images/skin/database_save.png
rename to myapp/grails-app/assets/images/skin/database_save.png
diff --git a/myplugin/grails-app/assets/images/skin/database_table.png b/myapp/grails-app/assets/images/skin/database_table.png
similarity index 100%
rename from myplugin/grails-app/assets/images/skin/database_table.png
rename to myapp/grails-app/assets/images/skin/database_table.png
diff --git a/myplugin/grails-app/assets/images/skin/exclamation.png b/myapp/grails-app/assets/images/skin/exclamation.png
similarity index 100%
rename from myplugin/grails-app/assets/images/skin/exclamation.png
rename to myapp/grails-app/assets/images/skin/exclamation.png
diff --git a/myplugin/grails-app/assets/images/skin/house.png b/myapp/grails-app/assets/images/skin/house.png
similarity index 100%
rename from myplugin/grails-app/assets/images/skin/house.png
rename to myapp/grails-app/assets/images/skin/house.png
diff --git a/myplugin/grails-app/assets/images/skin/information.png b/myapp/grails-app/assets/images/skin/information.png
similarity index 100%
rename from myplugin/grails-app/assets/images/skin/information.png
rename to myapp/grails-app/assets/images/skin/information.png
diff --git a/myplugin/grails-app/assets/images/skin/sorted_asc.gif b/myapp/grails-app/assets/images/skin/sorted_asc.gif
similarity index 100%
rename from myplugin/grails-app/assets/images/skin/sorted_asc.gif
rename to myapp/grails-app/assets/images/skin/sorted_asc.gif
diff --git a/myplugin/grails-app/assets/images/skin/sorted_desc.gif b/myapp/grails-app/assets/images/skin/sorted_desc.gif
similarity index 100%
rename from myplugin/grails-app/assets/images/skin/sorted_desc.gif
rename to myapp/grails-app/assets/images/skin/sorted_desc.gif
diff --git a/myplugin/grails-app/assets/images/slack.svg b/myapp/grails-app/assets/images/slack.svg
similarity index 100%
rename from myplugin/grails-app/assets/images/slack.svg
rename to myapp/grails-app/assets/images/slack.svg
diff --git a/myplugin/grails-app/assets/images/spinner.gif b/myapp/grails-app/assets/images/spinner.gif
similarity index 100%
rename from myplugin/grails-app/assets/images/spinner.gif
rename to myapp/grails-app/assets/images/spinner.gif
diff --git a/myplugin/grails-app/assets/javascripts/application.js b/myapp/grails-app/assets/javascripts/application.js
similarity index 100%
rename from myplugin/grails-app/assets/javascripts/application.js
rename to myapp/grails-app/assets/javascripts/application.js
diff --git a/myplugin/grails-app/assets/javascripts/bootstrap.bundle.js b/myapp/grails-app/assets/javascripts/bootstrap.bundle.js
similarity index 100%
rename from myplugin/grails-app/assets/javascripts/bootstrap.bundle.js
rename to myapp/grails-app/assets/javascripts/bootstrap.bundle.js
diff --git a/myplugin/grails-app/assets/javascripts/bootstrap.bundle.js.map b/myapp/grails-app/assets/javascripts/bootstrap.bundle.js.map
similarity index 100%
rename from myplugin/grails-app/assets/javascripts/bootstrap.bundle.js.map
rename to myapp/grails-app/assets/javascripts/bootstrap.bundle.js.map
diff --git a/myplugin/grails-app/assets/javascripts/bootstrap.bundle.min.js b/myapp/grails-app/assets/javascripts/bootstrap.bundle.min.js
similarity index 100%
rename from myplugin/grails-app/assets/javascripts/bootstrap.bundle.min.js
rename to myapp/grails-app/assets/javascripts/bootstrap.bundle.min.js
diff --git a/myplugin/grails-app/assets/javascripts/bootstrap.bundle.min.js.map b/myapp/grails-app/assets/javascripts/bootstrap.bundle.min.js.map
similarity index 100%
rename from myplugin/grails-app/assets/javascripts/bootstrap.bundle.min.js.map
rename to myapp/grails-app/assets/javascripts/bootstrap.bundle.min.js.map
diff --git a/myplugin/grails-app/assets/javascripts/bootstrap.js b/myapp/grails-app/assets/javascripts/bootstrap.js
similarity index 100%
rename from myplugin/grails-app/assets/javascripts/bootstrap.js
rename to myapp/grails-app/assets/javascripts/bootstrap.js
diff --git a/myplugin/grails-app/assets/javascripts/bootstrap.js.map b/myapp/grails-app/assets/javascripts/bootstrap.js.map
similarity index 100%
rename from myplugin/grails-app/assets/javascripts/bootstrap.js.map
rename to myapp/grails-app/assets/javascripts/bootstrap.js.map
diff --git a/myplugin/grails-app/assets/javascripts/bootstrap.min.js b/myapp/grails-app/assets/javascripts/bootstrap.min.js
similarity index 100%
rename from myplugin/grails-app/assets/javascripts/bootstrap.min.js
rename to myapp/grails-app/assets/javascripts/bootstrap.min.js
diff --git a/myplugin/grails-app/assets/javascripts/bootstrap.min.js.map b/myapp/grails-app/assets/javascripts/bootstrap.min.js.map
similarity index 100%
rename from myplugin/grails-app/assets/javascripts/bootstrap.min.js.map
rename to myapp/grails-app/assets/javascripts/bootstrap.min.js.map
diff --git a/myplugin/grails-app/assets/javascripts/jquery-3.5.1.js b/myapp/grails-app/assets/javascripts/jquery-3.5.1.js
similarity index 100%
rename from myplugin/grails-app/assets/javascripts/jquery-3.5.1.js
rename to myapp/grails-app/assets/javascripts/jquery-3.5.1.js
diff --git a/myplugin/grails-app/assets/javascripts/jquery-3.5.1.min.js b/myapp/grails-app/assets/javascripts/jquery-3.5.1.min.js
similarity index 100%
rename from myplugin/grails-app/assets/javascripts/jquery-3.5.1.min.js
rename to myapp/grails-app/assets/javascripts/jquery-3.5.1.min.js
diff --git a/myplugin/grails-app/assets/javascripts/jquery-3.5.1.min.js.map b/myapp/grails-app/assets/javascripts/jquery-3.5.1.min.js.map
similarity index 100%
rename from myplugin/grails-app/assets/javascripts/jquery-3.5.1.min.js.map
rename to myapp/grails-app/assets/javascripts/jquery-3.5.1.min.js.map
diff --git a/myplugin/grails-app/assets/javascripts/popper.js b/myapp/grails-app/assets/javascripts/popper.js
similarity index 100%
rename from myplugin/grails-app/assets/javascripts/popper.js
rename to myapp/grails-app/assets/javascripts/popper.js
diff --git a/myplugin/grails-app/assets/javascripts/popper.min.js b/myapp/grails-app/assets/javascripts/popper.min.js
similarity index 100%
rename from myplugin/grails-app/assets/javascripts/popper.min.js
rename to myapp/grails-app/assets/javascripts/popper.min.js
diff --git a/myplugin/grails-app/assets/javascripts/popper.min.js.map b/myapp/grails-app/assets/javascripts/popper.min.js.map
similarity index 100%
rename from myplugin/grails-app/assets/javascripts/popper.min.js.map
rename to myapp/grails-app/assets/javascripts/popper.min.js.map
diff --git a/myplugin/grails-app/assets/stylesheets/application.css b/myapp/grails-app/assets/stylesheets/application.css
similarity index 100%
rename from myplugin/grails-app/assets/stylesheets/application.css
rename to myapp/grails-app/assets/stylesheets/application.css
diff --git a/myplugin/grails-app/assets/stylesheets/bootstrap-grid.css b/myapp/grails-app/assets/stylesheets/bootstrap-grid.css
similarity index 100%
rename from myplugin/grails-app/assets/stylesheets/bootstrap-grid.css
rename to myapp/grails-app/assets/stylesheets/bootstrap-grid.css
diff --git a/myplugin/grails-app/assets/stylesheets/bootstrap-grid.css.map b/myapp/grails-app/assets/stylesheets/bootstrap-grid.css.map
similarity index 100%
rename from myplugin/grails-app/assets/stylesheets/bootstrap-grid.css.map
rename to myapp/grails-app/assets/stylesheets/bootstrap-grid.css.map
diff --git a/myplugin/grails-app/assets/stylesheets/bootstrap-grid.min.css b/myapp/grails-app/assets/stylesheets/bootstrap-grid.min.css
similarity index 100%
rename from myplugin/grails-app/assets/stylesheets/bootstrap-grid.min.css
rename to myapp/grails-app/assets/stylesheets/bootstrap-grid.min.css
diff --git a/myplugin/grails-app/assets/stylesheets/bootstrap-grid.min.css.map b/myapp/grails-app/assets/stylesheets/bootstrap-grid.min.css.map
similarity index 100%
rename from myplugin/grails-app/assets/stylesheets/bootstrap-grid.min.css.map
rename to myapp/grails-app/assets/stylesheets/bootstrap-grid.min.css.map
diff --git a/myplugin/grails-app/assets/stylesheets/bootstrap-reboot.css b/myapp/grails-app/assets/stylesheets/bootstrap-reboot.css
similarity index 100%
rename from myplugin/grails-app/assets/stylesheets/bootstrap-reboot.css
rename to myapp/grails-app/assets/stylesheets/bootstrap-reboot.css
diff --git a/myplugin/grails-app/assets/stylesheets/bootstrap.css b/myapp/grails-app/assets/stylesheets/bootstrap.css
similarity index 100%
rename from myplugin/grails-app/assets/stylesheets/bootstrap.css
rename to myapp/grails-app/assets/stylesheets/bootstrap.css
diff --git a/myplugin/grails-app/assets/stylesheets/bootstrap.css.map b/myapp/grails-app/assets/stylesheets/bootstrap.css.map
similarity index 100%
rename from myplugin/grails-app/assets/stylesheets/bootstrap.css.map
rename to myapp/grails-app/assets/stylesheets/bootstrap.css.map
diff --git a/myplugin/grails-app/assets/stylesheets/bootstrap.min.css b/myapp/grails-app/assets/stylesheets/bootstrap.min.css
similarity index 100%
rename from myplugin/grails-app/assets/stylesheets/bootstrap.min.css
rename to myapp/grails-app/assets/stylesheets/bootstrap.min.css
diff --git a/myplugin/grails-app/assets/stylesheets/bootstrap.min.css.map b/myapp/grails-app/assets/stylesheets/bootstrap.min.css.map
similarity index 100%
rename from myplugin/grails-app/assets/stylesheets/bootstrap.min.css.map
rename to myapp/grails-app/assets/stylesheets/bootstrap.min.css.map
diff --git a/myplugin/grails-app/assets/stylesheets/errors.css b/myapp/grails-app/assets/stylesheets/errors.css
similarity index 100%
rename from myplugin/grails-app/assets/stylesheets/errors.css
rename to myapp/grails-app/assets/stylesheets/errors.css
diff --git a/myplugin/grails-app/assets/stylesheets/grails.css b/myapp/grails-app/assets/stylesheets/grails.css
similarity index 100%
rename from myplugin/grails-app/assets/stylesheets/grails.css
rename to myapp/grails-app/assets/stylesheets/grails.css
diff --git a/myplugin/grails-app/assets/stylesheets/main.css b/myapp/grails-app/assets/stylesheets/main.css
similarity index 100%
rename from myplugin/grails-app/assets/stylesheets/main.css
rename to myapp/grails-app/assets/stylesheets/main.css
diff --git a/myplugin/grails-app/assets/stylesheets/mobile.css b/myapp/grails-app/assets/stylesheets/mobile.css
similarity index 100%
rename from myplugin/grails-app/assets/stylesheets/mobile.css
rename to myapp/grails-app/assets/stylesheets/mobile.css
diff --git a/myplugin/grails-app/conf/application.yml b/myapp/grails-app/conf/application.yml
similarity index 99%
rename from myplugin/grails-app/conf/application.yml
rename to myapp/grails-app/conf/application.yml
index 2f681f6..1190eb0 100644
--- a/myplugin/grails-app/conf/application.yml
+++ b/myapp/grails-app/conf/application.yml
@@ -2,7 +2,7 @@
 grails:
     profile: web
     codegen:
-        defaultPackage: myplugin
+        defaultPackage: myapp
     gorm:
         reactor:
             # Whether to translate GORM events into Reactor events
diff --git a/myplugin/grails-app/conf/logback.xml b/myapp/grails-app/conf/logback.xml
similarity index 100%
rename from myplugin/grails-app/conf/logback.xml
rename to myapp/grails-app/conf/logback.xml
diff --git a/myplugin/grails-app/conf/spring/resources.groovy b/myapp/grails-app/conf/spring/resources.groovy
similarity index 100%
rename from myplugin/grails-app/conf/spring/resources.groovy
rename to myapp/grails-app/conf/spring/resources.groovy
diff --git a/myplugin/grails-app/controllers/myplugin/UrlMappings.groovy b/myapp/grails-app/controllers/myapp/UrlMappings.groovy
similarity index 94%
rename from myplugin/grails-app/controllers/myplugin/UrlMappings.groovy
rename to myapp/grails-app/controllers/myapp/UrlMappings.groovy
index 90d38d4..1be4f0c 100644
--- a/myplugin/grails-app/controllers/myplugin/UrlMappings.groovy
+++ b/myapp/grails-app/controllers/myapp/UrlMappings.groovy
@@ -1,4 +1,4 @@
-package myplugin
+package myapp
 
 class UrlMappings {
 
diff --git a/myplugin/grails-app/i18n/messages.properties b/myapp/grails-app/i18n/messages.properties
similarity index 100%
rename from myplugin/grails-app/i18n/messages.properties
rename to myapp/grails-app/i18n/messages.properties
diff --git a/myplugin/grails-app/i18n/messages_cs.properties b/myapp/grails-app/i18n/messages_cs.properties
similarity index 100%
rename from myplugin/grails-app/i18n/messages_cs.properties
rename to myapp/grails-app/i18n/messages_cs.properties
diff --git a/myplugin/grails-app/i18n/messages_da.properties b/myapp/grails-app/i18n/messages_da.properties
similarity index 100%
rename from myplugin/grails-app/i18n/messages_da.properties
rename to myapp/grails-app/i18n/messages_da.properties
diff --git a/myplugin/grails-app/i18n/messages_de.properties b/myapp/grails-app/i18n/messages_de.properties
similarity index 100%
rename from myplugin/grails-app/i18n/messages_de.properties
rename to myapp/grails-app/i18n/messages_de.properties
diff --git a/myplugin/grails-app/i18n/messages_es.properties b/myapp/grails-app/i18n/messages_es.properties
similarity index 100%
rename from myplugin/grails-app/i18n/messages_es.properties
rename to myapp/grails-app/i18n/messages_es.properties
diff --git a/myplugin/grails-app/i18n/messages_fr.properties b/myapp/grails-app/i18n/messages_fr.properties
similarity index 100%
rename from myplugin/grails-app/i18n/messages_fr.properties
rename to myapp/grails-app/i18n/messages_fr.properties
diff --git a/myplugin/grails-app/i18n/messages_it.properties b/myapp/grails-app/i18n/messages_it.properties
similarity index 100%
rename from myplugin/grails-app/i18n/messages_it.properties
rename to myapp/grails-app/i18n/messages_it.properties
diff --git a/myplugin/grails-app/i18n/messages_ja.properties b/myapp/grails-app/i18n/messages_ja.properties
similarity index 100%
rename from myplugin/grails-app/i18n/messages_ja.properties
rename to myapp/grails-app/i18n/messages_ja.properties
diff --git a/myplugin/grails-app/i18n/messages_nb.properties b/myapp/grails-app/i18n/messages_nb.properties
similarity index 100%
rename from myplugin/grails-app/i18n/messages_nb.properties
rename to myapp/grails-app/i18n/messages_nb.properties
diff --git a/myplugin/grails-app/i18n/messages_nl.properties b/myapp/grails-app/i18n/messages_nl.properties
similarity index 100%
rename from myplugin/grails-app/i18n/messages_nl.properties
rename to myapp/grails-app/i18n/messages_nl.properties
diff --git a/myplugin/grails-app/i18n/messages_pl.properties b/myapp/grails-app/i18n/messages_pl.properties
similarity index 100%
rename from myplugin/grails-app/i18n/messages_pl.properties
rename to myapp/grails-app/i18n/messages_pl.properties
diff --git a/myplugin/grails-app/i18n/messages_pt_BR.properties b/myapp/grails-app/i18n/messages_pt_BR.properties
similarity index 100%
rename from myplugin/grails-app/i18n/messages_pt_BR.properties
rename to myapp/grails-app/i18n/messages_pt_BR.properties
diff --git a/myplugin/grails-app/i18n/messages_pt_PT.properties b/myapp/grails-app/i18n/messages_pt_PT.properties
similarity index 100%
rename from myplugin/grails-app/i18n/messages_pt_PT.properties
rename to myapp/grails-app/i18n/messages_pt_PT.properties
diff --git a/myplugin/grails-app/i18n/messages_ru.properties b/myapp/grails-app/i18n/messages_ru.properties
similarity index 100%
rename from myplugin/grails-app/i18n/messages_ru.properties
rename to myapp/grails-app/i18n/messages_ru.properties
diff --git a/myplugin/grails-app/i18n/messages_sk.properties b/myapp/grails-app/i18n/messages_sk.properties
similarity index 100%
rename from myplugin/grails-app/i18n/messages_sk.properties
rename to myapp/grails-app/i18n/messages_sk.properties
diff --git a/myplugin/grails-app/i18n/messages_sv.properties b/myapp/grails-app/i18n/messages_sv.properties
similarity index 100%
rename from myplugin/grails-app/i18n/messages_sv.properties
rename to myapp/grails-app/i18n/messages_sv.properties
diff --git a/myplugin/grails-app/i18n/messages_th.properties b/myapp/grails-app/i18n/messages_th.properties
similarity index 100%
rename from myplugin/grails-app/i18n/messages_th.properties
rename to myapp/grails-app/i18n/messages_th.properties
diff --git a/myplugin/grails-app/i18n/messages_zh_CN.properties b/myapp/grails-app/i18n/messages_zh_CN.properties
similarity index 100%
rename from myplugin/grails-app/i18n/messages_zh_CN.properties
rename to myapp/grails-app/i18n/messages_zh_CN.properties
diff --git a/myplugin/grails-app/init/myplugin/Application.groovy b/myapp/grails-app/init/myapp/Application.groovy
similarity index 93%
rename from myplugin/grails-app/init/myplugin/Application.groovy
rename to myapp/grails-app/init/myapp/Application.groovy
index ace04f1..980180d 100644
--- a/myplugin/grails-app/init/myplugin/Application.groovy
+++ b/myapp/grails-app/init/myapp/Application.groovy
@@ -1,4 +1,4 @@
-package myplugin
+package myapp
 
 import grails.boot.GrailsApp
 import grails.boot.config.GrailsAutoConfiguration
diff --git a/myplugin/grails-app/init/myplugin/BootStrap.groovy b/myapp/grails-app/init/myapp/BootStrap.groovy
similarity index 83%
rename from myplugin/grails-app/init/myplugin/BootStrap.groovy
rename to myapp/grails-app/init/myapp/BootStrap.groovy
index e8ad49b..5cf149b 100644
--- a/myplugin/grails-app/init/myplugin/BootStrap.groovy
+++ b/myapp/grails-app/init/myapp/BootStrap.groovy
@@ -1,4 +1,4 @@
-package myplugin
+package myapp
 
 class BootStrap {
 
diff --git a/myplugin/grails-app/views/error.gsp b/myapp/grails-app/views/error.gsp
similarity index 100%
rename from myplugin/grails-app/views/error.gsp
rename to myapp/grails-app/views/error.gsp
diff --git a/myplugin/grails-app/views/index.gsp b/myapp/grails-app/views/index.gsp
similarity index 100%
rename from myplugin/grails-app/views/index.gsp
rename to myapp/grails-app/views/index.gsp
diff --git a/myplugin/grails-app/views/layouts/main.gsp b/myapp/grails-app/views/layouts/main.gsp
similarity index 100%
rename from myplugin/grails-app/views/layouts/main.gsp
rename to myapp/grails-app/views/layouts/main.gsp
diff --git a/myplugin/grails-app/views/notFound.gsp b/myapp/grails-app/views/notFound.gsp
similarity index 100%
rename from myplugin/grails-app/views/notFound.gsp
rename to myapp/grails-app/views/notFound.gsp
diff --git a/myplugin/grails-wrapper.jar b/myapp/grails-wrapper.jar
similarity index 100%
rename from myplugin/grails-wrapper.jar
rename to myapp/grails-wrapper.jar
diff --git a/myplugin/grailsw b/myapp/grailsw
similarity index 100%
rename from myplugin/grailsw
rename to myapp/grailsw
diff --git a/myplugin/grailsw.bat b/myapp/grailsw.bat
similarity index 100%
rename from myplugin/grailsw.bat
rename to myapp/grailsw.bat
diff --git a/myapp/settings.gradle b/myapp/settings.gradle
new file mode 100644
index 0000000..6d35791
--- /dev/null
+++ b/myapp/settings.gradle
@@ -0,0 +1 @@
+rootProject.name = 'myapp'
diff --git a/myplugin/src/integration-test/resources/GebConfig.groovy b/myapp/src/integration-test/resources/GebConfig.groovy
similarity index 100%
rename from myplugin/src/integration-test/resources/GebConfig.groovy
rename to myapp/src/integration-test/resources/GebConfig.groovy
diff --git a/myplugin/settings.gradle b/myplugin/settings.gradle
deleted file mode 100644
index 8d5eb67..0000000
--- a/myplugin/settings.gradle
+++ /dev/null
@@ -1 +0,0 @@
-rootProject.name = 'myplugin'
