diff --git a/myplugin/gradle.properties b/myplugin/gradle.properties
index 377f3173..8b89f17c 100644
--- a/myplugin/gradle.properties
+++ b/myplugin/gradle.properties
@@ -1,7 +1,7 @@
-grailsVersion=5.3.6
-grailsGradlePluginVersion=5.3.1
+grailsVersion=6.1.0
+grailsGradlePluginVersion=6.1.0
 groovyVersion=3.0.11
-gorm.version=7.3.4
+gorm.version=8.0.2
 org.gradle.daemon=true
 org.gradle.parallel=true
 org.gradle.jvmargs=-Dfile.encoding=UTF-8 -Xmx1024M
diff --git a/myplugin/src/main/groovy/myplugin/MypluginGrailsPlugin.groovy b/myplugin/src/main/groovy/myplugin/MypluginGrailsPlugin.groovy
index 15bd8f91..ff3cd69f 100644
--- a/myplugin/src/main/groovy/myplugin/MypluginGrailsPlugin.groovy
+++ b/myplugin/src/main/groovy/myplugin/MypluginGrailsPlugin.groovy
@@ -5,7 +5,7 @@ import grails.plugins.*
 class MypluginGrailsPlugin extends Plugin {
 
     // the version or versions of Grails the plugin is designed for
-    def grailsVersion = "5.3.6 > *"
+    def grailsVersion = "6.1.0 > *"
     // resources that are excluded from plugin packaging
     def pluginExcludes = [
         "grails-app/views/error.gsp"
