diff --git a/myplugin/gradle.properties b/myplugin/gradle.properties
index 2177c2cd..05aa7c28 100644
--- a/myplugin/gradle.properties
+++ b/myplugin/gradle.properties
@@ -1,7 +1,7 @@
-grailsVersion=6.0.0
-grailsGradlePluginVersion=6.0.0
+grailsVersion=5.3.3
+grailsGradlePluginVersion=5.3.0
 groovyVersion=3.0.11
-gorm.version=8.0.0
+gorm.version=7.3.3
 org.gradle.daemon=true
 org.gradle.parallel=true
 org.gradle.jvmargs=-Dfile.encoding=UTF-8 -Xmx1024M
diff --git a/myplugin/src/main/groovy/myplugin/MypluginGrailsPlugin.groovy b/myplugin/src/main/groovy/myplugin/MypluginGrailsPlugin.groovy
index 2c9d78be..419962bb 100644
--- a/myplugin/src/main/groovy/myplugin/MypluginGrailsPlugin.groovy
+++ b/myplugin/src/main/groovy/myplugin/MypluginGrailsPlugin.groovy
@@ -5,7 +5,7 @@ import grails.plugins.*
 class MypluginGrailsPlugin extends Plugin {
 
     // the version or versions of Grails the plugin is designed for
-    def grailsVersion = "6.0.0 > *"
+    def grailsVersion = "5.3.3 > *"
     // resources that are excluded from plugin packaging
     def pluginExcludes = [
         "grails-app/views/error.gsp"
