diff --git a/myapp/.gitignore b/myapp/.gitignore
index dcdf0171..cc025997 100644
--- a/myapp/.gitignore
+++ b/myapp/.gitignore
@@ -4,7 +4,6 @@ build/
 !gradle/wrapper/gradle-wrapper.jar
 !**/src/main/**/build/
 !**/src/test/**/build/
-!**/src/integration-test/**/build/
 
 ### STS ###
 .apt_generated
@@ -17,7 +16,6 @@ build/
 bin/
 !**/src/main/**/bin/
 !**/src/test/**/bin/
-!**/src/integration-test/**/bin/
 
 ### IntelliJ IDEA ###
 .idea
@@ -27,7 +25,6 @@ bin/
 out/
 !**/src/main/**/out/
 !**/src/test/**/out/
-!**/src/integration-test/**/out/
 
 ### NetBeans ###
 /nbproject/private/
diff --git a/myapp/README.md b/myapp/README.md
index ddf8b035..75b2e8f1 100644
--- a/myapp/README.md
+++ b/myapp/README.md
@@ -1,7 +1,7 @@
-## Grails 7.0.0-M3 Documentation
+## Grails 7.0.0-M1 Documentation
 
-- [User Guide](https://docs.grails.org/7.0.0-M3/guide/index.html)
-- [API Reference](https://docs.grails.org/7.0.0-M3/api/index.html)
+- [User Guide](https://docs.grails.org/7.0.0-M1/guide/index.html)
+- [API Reference](https://docs.grails.org/7.0.0-M1/api/index.html)
 - [Grails Guides](https://guides.grails.org/index.html)
 ---
 
diff --git a/myapp/build.gradle b/myapp/build.gradle
index 5b762815..b830296b 100644
--- a/myapp/build.gradle
+++ b/myapp/build.gradle
@@ -4,10 +4,9 @@ buildscript {
         mavenCentral()
     }
     dependencies { // Not Published to Gradle Plugin Portal
-        classpath platform("org.grails:grails-bom:$grailsVersion")
-        classpath "org.grails:grails-gradle-plugin"
-        classpath "org.grails.plugins:hibernate5"
-        classpath "org.grails.plugins:views-gradle"
+        classpath "org.grails:grails-gradle-plugin:7.0.0-M3"
+        classpath "org.grails.plugins:hibernate5:9.0.0-M2"
+        classpath "org.grails.plugins:views-gradle:4.0.0-M1"
     }
 }
 
@@ -52,7 +51,7 @@ dependencies {
     implementation "org.springframework.boot:spring-boot-starter-validation"
     console "org.grails:grails-console"
     runtimeOnly "com.h2database:h2"
-    runtimeOnly "com.zaxxer:HikariCP"
+    runtimeOnly "org.apache.tomcat:tomcat-jdbc"
     testImplementation "org.grails:grails-gorm-testing-support"
     testImplementation "org.grails:views-json-testing-support"
     testImplementation "org.spockframework:spock-core"
@@ -63,4 +62,3 @@ compileJava.options.release = 17
 tasks.withType(Test) {
     useJUnitPlatform()
 }
-
diff --git a/myapp/gradle.properties b/myapp/gradle.properties
index a0d44a97..bfeddf7f 100644
--- a/myapp/gradle.properties
+++ b/myapp/gradle.properties
@@ -1,4 +1,5 @@
-grailsVersion=7.0.0-M3
+grailsVersion=7.0.0-M1
+grailsGradlePluginVersion=7.0.0-M3
 version=0.1
 org.gradle.caching=true
 org.gradle.daemon=true
diff --git a/myapp/grails-app/conf/application.yml b/myapp/grails-app/conf/application.yml
index d6976c43..f5da6dfd 100644
--- a/myapp/grails-app/conf/application.yml
+++ b/myapp/grails-app/conf/application.yml
@@ -4,9 +4,6 @@ info:
     version: '@info.app.version@'
     grailsVersion: '@info.app.grailsVersion@'
 grails:
-  codegen:
-    defaultPackage: myapp
-  profile: rest-api
   mime:
     disable:
       accept:
@@ -36,6 +33,9 @@ grails:
   views:
     default:
       codec: html
+  codegen:
+    defaultPackage: myapp
+  profile: rest-api
 dataSource:
   driverClassName: org.h2.Driver
   username: sa
@@ -55,6 +55,24 @@ environments:
     dataSource:
       dbCreate: none
       url: jdbc:h2:./prodDb;LOCK_TIMEOUT=10000;DB_CLOSE_ON_EXIT=FALSE
+      properties:
+        jmxEnabled: true
+        initialSize: 5
+        maxActive: 50
+        minIdle: 5
+        maxIdle: 25
+        maxWait: 10000
+        maxAge: 600000
+        timeBetweenEvictionRunsMillis: 5000
+        minEvictableIdleTimeMillis: 60000
+        validationQuery: SELECT 1
+        validationQueryTimeout: 3
+        validationInterval: 15000
+        testOnBorrow: true
+        testWhileIdle: true
+        testOnReturn: false
+        jdbcInterceptors: ConnectionState
+        defaultTransactionIsolation: 2
 hibernate:
   cache:
     queries: false
diff --git a/myapp/grails-app/init/myapp/Application.groovy b/myapp/grails-app/init/myapp/Application.groovy
index 6c9292cf..ca9e131e 100644
--- a/myapp/grails-app/init/myapp/Application.groovy
+++ b/myapp/grails-app/init/myapp/Application.groovy
@@ -3,6 +3,7 @@ package myapp
 import grails.boot.GrailsApp
 import grails.boot.config.GrailsAutoConfiguration
 import groovy.transform.CompileStatic
+import grails.plugins.metadata.*
 
 @CompileStatic
 class Application extends GrailsAutoConfiguration {
diff --git a/myapp/grails-app/init/myapp/BootStrap.groovy b/myapp/grails-app/init/myapp/BootStrap.groovy
index fff95730..4e73b71c 100644
--- a/myapp/grails-app/init/myapp/BootStrap.groovy
+++ b/myapp/grails-app/init/myapp/BootStrap.groovy
@@ -1,15 +1,9 @@
 package myapp
 
-import jakarta.servlet.ServletContext
-
 class BootStrap {
 
-    ServletContext servletContext
-
-    def init = {
+    def init = { servletContext ->
     }
-
     def destroy = {
     }
-
 }
\ No newline at end of file
